spring-cloud-gcp-data-spanner/src/main/java/org/springframework/cloud/gcp/data/spanner/core/convert/ConverterAwareMappingSpannerEntityWriter.java

```diff
@@ -321,7 +319,9 @@ private boolean attemptSetIterableValue(Iterable<Object> value,
 			return false;
 		}
 		// We're just checking for the bind to have succeeded, we don't need to chain the result.
-		Object ignored = toMethod.apply(valueBinder, this.writeConverter.convert(value, targetType));
+		// Spanner allows binding of null values.
+		Object ignored = toMethod.apply(valueBinder,
```

Comment:
Why are we capturing the `ignored` method return value?

Reply:
Let me check, but if I recall it was for a compiler warning? I'll check

Reply:
Ah it was this:
```
[ERROR] /usr/local/google/home/cccz/IdeaProjects/spring-cloud-gcp/spring-cloud-gcp-data-spanner/src/main/java/org/springframework/cloud/gcp/data/spanner/core/convert/ConverterAwareMappingSpannerEntityWriter.java:[323,31] [ReturnValueIgnored] Return value of this method must be used
```

That is what happens if the return value isn't stored to something.